frontend/www/js/omegaup/components/user/PreferencesEdit.test.ts

```diff
@@ -0,0 +1,67 @@
+import { shallowMount } from '@vue/test-utils';
+import { types } from '../../api_types';
+import user_Preferences_Edit from './PreferencesEdit.vue';
+
+const profile: types.UserProfileInfo = {
+  name: 'omegaUp admin',
+  classname: 'user-rank-unranked',
+  email: 'admin@omegaup.com',
+  username: 'omegaup',
+  verified: true,
+  hide_problem_tags: false,
+  is_private: false,
+  preferred_language: 'py2',
+  programming_languages: {
+    py2: 'python2',
+    rb: 'ruby',
+  },
+  rankinfo: {
+    name: 'Test',
+    problems_solved: 2,
+    rank: 1,
+  },
+  is_own_profile: true,
+  birth_date: new Date('1999-09-09'),
+  locale: 'es',
+  gender: 'decline',
+};
+
+describe('PreferencesEdit.vue', () => {
+  it('Should display user email', () => {
+    const wrapper = shallowMount(user_Preferences_Edit, {
+      propsData: { profile: profile },
```

Comment:
Acá con esto es suficiente:
```suggestion
      propsData: { profile },
```

Mismo caso para L39